tests/acceptance/brocfile-smoke-test-slow.js

```diff
@@ -43,6 +40,9 @@ describe('Acceptance: brocfile-smoke-test', function () {
   });
 
   it('a custom EmberENV in config/environment.js is used for window.EmberENV', async function () {
+    if (isExperimentEnabled('EMBROIDER')) {
+      this.skip();
+    }
```

Comment:
I think these will have to return, right? Otherwise we still run to completion (e.g. running the tests) only to mark them as skipped later.

Am I mising something?